src/binaryen-c.cpp

```diff
@@ -5451,7 +5451,9 @@ void BinaryenModuleRunPasses(BinaryenModuleRef module,
   PassRunner passRunner((Module*)module);
   passRunner.options = globalPassOptions;
   for (BinaryenIndex i = 0; i < numPasses; i++) {
-    passRunner.add(passes[i]);
+    passRunner.add(passes[i]), globalPassOptions.arguments.count(passes[i]) > 0
+                                 ? globalPassOptions.arguments[passes[i]]
+                                 : std::optional<std::string>();
```

Comment:
I don't understand this line... it looks like
```cpp
A, B ? C : D;
```
where `A` has an effect (it adds the pass) but `B` just counts something and `C` and `D` have no side effects, so this looks equivalent to
```cpp
A;
```
which is
```cpp
passRunner.add(passes[i])
```
Perhaps this was meant to be
```cpp
passRunner.add(passes[i], ....);
```
(rather than to use the comma operator after closing the `add`)?

Reply:
Of course, you're right, I messed up the braces. Fixed.